changelog/1676.feature.rst

```diff
@@ -1,3 +1,3 @@
-Added the `~plasmapy.formulary.collisions.CollisionFrequencies` class. This class can be used to calculate collision frequencies for two interacting plasmas.
+Added the `~plasmapy.formulary.collisions.SingleParticleCollisionFrequencies` class. This class can be used to calculate collision frequencies for two interacting plasmas.
```

Comment:
I don't think we should edit the existing changelog - just add lines to the changelog for this PR saying that we renamed the function. We want the changelogs to correspond to what was actually done in that PR for bookkeeping purposes.

Reply:
What should I do about the failing documentation build?